core/src/main/java/io/questdb/cutlass/auth/AuthUtils.java

```diff
@@ -24,19 +24,115 @@
 
 package io.questdb.cutlass.auth;
 
+import io.questdb.std.CharSequenceObjHashMap;
 import io.questdb.std.Chars;
+import io.questdb.std.ThreadLocal;
 
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
 import java.math.BigInteger;
+import java.nio.ByteBuffer;
 import java.security.*;
 import java.security.interfaces.ECKey;
 import java.security.spec.*;
 import java.util.Base64;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public final class AuthUtils {
     public static final String EC_ALGORITHM = "EC";
     public static final String EC_CURVE = "secp256r1";
+    // there are 2 types of signatures:
+    // 1. sigP1363 format is 64 bytes
+    // 2. sigDER format is 72 bytes
```

Comment:
intellij shows 'unused field' warning . 
How about adding @SuppressWarnings or using it to compute MAX_SIGNATURE_LENGTH_BASE64  ?

Reply:
how about now? do you reckon it's better?

Reply:
👍